Packs/AutoFocus/Integrations/AutofocusV2/AutofocusV2.yml

```
@@ -407,8 +407,9 @@ script:
     deprecated: false
     description: Searches for sessions in AutoFocus. To view results, run the autofocus-sessions-search-results
       command with the returned AF Cookie. The AF Cookie expires 120 seconds after
-      the search completes. `Autofocus Query Samples, Sessions and Tags` Playbook
-      is recommended for querying and polling.
+      the search completes. Note that if you pass more than 100 IPs, it may not work when searching the result with the returned AF cookie,
+      you will need to iterate the IPs and search for each AF cookie, this is due to an AF limitation.
```

```suggestion
      the search completes. Note that if you pass more than 100 IPs, it may not work when searching the result with the returned AF cookie, due to an API limitation.
      In this case, you will need to iterate over the IPs and search for each AF cookie.
```
I'm just not 100% what you meant by "and search for each AF cookie", can you clarify?

---

The bug was when the user tried to search with the returned AF cookie, what I'm saying is that if the user will have more than 100 IPs, he will need to run the command for each 100 IPs and search the result with each AF cookie.

---

Pushed a improve with the CR fix.